Dockerfile

```diff
@@ -35,5 +38,7 @@ USER mumble
 # Set volumes
 VOLUME /etc/mumble
 
-# Default command
-CMD ["/opt/mumble/murmur.x86", "-fg", "-ini", "/etc/mumble/config.ini"]
+# Entrypoint
+COPY entrypoint.sh /app/entrypoint.sh
+ENTRYPOINT [ "/app/entrypoint.sh" ]
```

Comment:
`ENTRYPOINT` is only used when the container can be run as an executable. This should be part of the `CMD` instruction instead. Then the shell script would call `murmur.x86` with `exec`.

See the following for more information on the difference between `ENTRYPOINT` and `CMD`:
  - https://docs.docker.com/engine/reference/builder/#entrypoint
  - https://docs.docker.com/engine/reference/builder/#cmd

Reply:
I thought it was the intention to allow running this container as an executable too. Thats the reason I tried to separate `ENTRYPOINT` and `CMD`.

I am not sure though exactly what you mean by
> This should be part of the CMD instruction instead

because your suggestion for the `CMD` line does not contain the `"/app/entrypoint.sh"` of the entrypoint, could you please clarify?

Reply:
No, this container does not run as an executable. For an example of a container that runs as an executable check out https://github.com/PHLAK/docker-youtube-dl